web-app/js/plugins/jquery.custom.draggable.js

```diff
@@ -0,0 +1,43 @@
+// src: https://css-tricks.com/snippets/jquery/draggable-without-jquery-ui/
```

Comment:
@rahulparekh why did we customize `draggable` rather than using `jquery-ui` `draggable`?